### DISCUSSION TITLE
Crypt data on db

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6836] [General]
Hi,

I need to save some data encrypted on db and decrypt them before use

Is there some best practices in serenity?

Thank you,
Elena.

**Comment by VictorTomaili** [comment]
field behaviors 
https://serenity.is/docs/tutorials/multi_tenancy/using_serenity_service_behaviors

https://github.com/serenity-is/common-features/blob/aed9f429d8711b88a727b703f26c24434d5f9b94/src/Serenity.Demo.Northwind/Modules/Note/NotesBehavior.cs
